### ISSUE TITLE
Multiple high severity vulnerabilities warnings while installing Remix Blue stack

### ISSUE THREAD

**Issue by binajmen** [remix-run/remix#2373]
### What version of Remix are you using?

latest via `npx create-remix`

### Steps to Reproduce

Run `npx create-remix --template remix-run/blues-stack` in console

### Expected Behavior

No severe warnings

### Actual Behavior

```
➜  ~/Projects  npx create-remix --template remix-run/blues-stack
Need to install the following packages:
  create-remix
Ok to proceed? (y) y
💿 Welcome to Remix! Let's get you set up with a new project.

npm WARN deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.
? Where would you like to create your app? ./my-project
? Do you want me to run `npm install`? Yes
npm WARN deprecated source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

> postinstall
> remix setup node

Successfully setup Remix for node.

added 1212 packages, and audited 1213 packages in 53s

230 packages are looking for funding
  run `npm fund` for details

4 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
💿 Running remix.init script
Setup is almost complete. Follow these steps to finish initialization:

- Start the database:
  npm run docker

- Run setup:
  npm run setup

- You're now ready to rock and roll 🤘
  npm run dev
💿 That's it! `cd` into "/Users/benjamin/Projects/my-project" and check the README for development and deploy instructions!
➜  ~/Projects  npm audit fix
npm ERR! code ENOLOCK
npm ERR! audit This command requires an existing lockfile.
npm ERR! audit Try creating one first with: npm i --package-lock-only
npm ERR! audit Original error: loadVirtual requires existing shrinkwrap file

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/benjamin/.npm/_logs/2022-03-18T09_40_34_017Z-debug-0.log
➜  ~/Projects  cd my-project
➜  ~/Projects/my-project  npm audit    
# npm audit report

trim-newlines  <3.0.1
Severity: high
Regular Expression Denial of Service in trim-newlines - https://github.com/advisories/GHSA-7p7h-4mm5-852v
fix available via `npm audit fix --force`
Will install @remix-run/dev@1.2.3, which is a breaking change
node_modules/chalk-animation/node_modules/trim-newlines
  meow  3.4.0 - 5.0.0
  Depends on vulnerable versions of trim-newlines
  node_modules/chalk-animation/node_modules/meow
    chalk-animation  >=1.4.0
    Depends on vulnerable versions of meow
    node_modules/chalk-animation
      @remix-run/dev  <=0.0.0-experimental-fde68a7 || >=1.3.0-pre.1
      Depends on vulnerable versions of chalk-animation
      node_modules/@remix-run/dev

4 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force
```

**Comment by binajmen** [comment]
Moved to blue-stack repo: https://github.com/remix-run/blues-stack/issues/24

**Comment by salper** [comment]
FTR: it's coming from [here](https://github.com/bokub/chalk-animation/issues/20).

**Comment by isaacs** [comment]
Anyone finding this, you can add this to your `package.json` file:

```json
  "overrides": {
    "meow@4": "7"
  }
```

and then run `npm update`, and the problem is gone.

The breaking changes in meow 4 -> 7 were mostly just dropping old node versions and changing `hard-rejection` to `loud-rejection`, but chalk-animations doesn't use that feature anyway.

**Comment by MichaelDeBoey** [comment]
https://github.com/remix-run/remix/pull/2547 is merged, so it will be released when `3.4.0` gets released.
Once that's done, we'll update the `package-lock.json` of all stacks, which should fix these warnings

**Comment by bokub** [comment]
In case you need `chalk-animation` again, the issues have been fixed in version `2.0.0`, and all dependencies are now up to date.

By the way, none of the security alerts were relevant because the affected functions / modules were not used in `chalk-animation`, but I agree that the alerts were annoying.

Have a good day! :+1:
